osbenchmark/builder/launchers/docker_launcher.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import os
+
+from osbenchmark import time, telemetry
+from osbenchmark.builder import cluster
+from osbenchmark.builder.launchers.launcher import Launcher
+from osbenchmark.exceptions import LaunchError, ExecutorError
+
+
+class DockerLauncher(Launcher):
+    # May download a Docker image and that can take some time
```

Comment:
Might worth adding a 1-line documentation about ``DockerLauncher``, like the ``ProcessLauncher`` below.

Reply:
I'm not sure that I agree with documenting interface implementations. It adds burden when the implementation changes. I'd rather rely on documentation in the interface definition that users can refer to for more context, then have the code in the implementation self-document with well written variable and method names. 

For this case in particular, I think DockerLauncher is relatively self-explanatory